packages/braid-design-system/src/lib/themes/wireframe/tokens.ts

```diff
@@ -50,6 +50,7 @@ const getHoverColor = (x: string) =>
 const tokens: BraidTokens = {
   name: 'wireframe',
   displayName: 'Wireframe',
+  legacy: false,
```

Comment:
Does this mean once `apac` is dropped this theme will become the wireframe for `seekJobs`?